project-base/config/routes/fm_elfinder.yaml

```diff
@@ -1,2 +1,2 @@
 fm_elfinder:
-    resource: "@FMElfinderBundle/Resources/config/routing.yml"
+    resource: "@FMElfinderBundle/Resources/config/routing.yaml"
```

Comment:
file was renamed to yaml in version `10.0.1`, minimum required version is 10.0 according to composer.json and that version will not work.
I suggest to increase version in composer.json

Reply:
Done